### ISSUE TITLE
(Feature request) Support /metrics endpoint

### ISSUE THREAD

**Issue by perfectra1n** [TriliumNext/Notes#437]
### Describe feature

I think it would be cool to support a `/metrics` endpoint to gather data about a Trilium instance. We could still put it behind authN, this endpoint is common for tools like Prometheus that scrape information from various services to then display in Grafana. If we think this is a trash idea, I'll put it back in the 🗑.

**Comment by perfectra1n** [comment]
(This was inspired by [this issue](https://github.com/gethomepage/homepage/discussions/2312#discussioncomment-10479153) for a Trilium homepage widget)

- Something like a basic health check (does `SELECT 1` against the SQLite DB succeed)
- # of notes
- # of revisions
- Trilium version
- Sync version
- DB version
- DB size?

I believe are the first things that come to mind...

**Comment by perfectra1n** [comment]
Yeah I think it's slightly related to that issue @meichthys but I think the majority of those requests were for some form of "in app" visibility / metrics, while this would be "outside" of the app in a sense...

**Comment by perfectra1n** [comment]
As a really silly example of something similar, here's my Nextcloud dashboard
![image](https://github.com/user-attachments/assets/adbbe68e-cf42-42d0-9dc6-f7f9474e0f90)

**Comment by pano9000** [comment]
> * Trilium version
> 
> * Sync version
> 
> * DB version

if this is already required: that type of information can be checked via `/api/app-info` already.

I do like the additional idea of metrics though like the ones you mentioned above, so +1 from my side

**Comment by perfectra1n** [comment]
Implemented in v0.94.0 :)
